src/app/(admin)/services/providers/staff/page.tsx

```
@@ -0,0 +1,232 @@
+                if (contentType && contentType.includes('application/json')) {
+                    errorData = await res.json();
+                }
+                setFormError(errorData.error || 'Failed to save staff');
+                toast({ title: 'Error', description: errorData.error || 'Failed to save staff', variant: 'destructive' });
+                return;
+            }
+            // Refresh staff list
+            const refreshed = await fetch(`/api/providers/${staffProviderId}/staff`);
+            let json: { data?: StaffMember[] } = {};
+            const contentType = refreshed.headers.get('content-type');
+            if (contentType && contentType.includes('application/json')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)